dags/tower_metrics_dag.py

```diff
@@ -1,18 +1,27 @@
 from datetime import datetime
 
 from airflow.decorators import dag
+from airflow.models.param import Param
 
 from dag_content.tower_metrics_content import *
 
+dag_params = {
+    "user_list": Param(
+        "bwmac,thomas.yu",  # no support for python lists
+        type="string",
+    )
+}
+
 
 @dag(
-    schedule_interval="@weekly",
+    schedule_interval="0 15 * * 5",  # Fridays at 3PM PST
```

Comment:
Will this be UTC time?

Reply:
I did forget to adjust for UTC, now I think it is correct